### ISSUE TITLE
Cannot convert SSD mobilenet v2 to CoreML

### ISSUE THREAD

**Issue by zampnrs** [tf-coreml/tf-coreml#393]
I've trained my custom data on Tensorflow using the **ssd_mobilenet_v2_quantized_coco** from [tensorflow model zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md). When I tried to convert the pb file to mlmodel I got the error: Graph has cycles, as the #27. In that issue someone said to use the Graph Transformation Tool from Tensorflow.
**My questions: ** 
1- How can I use the Graph Transformation Tool to make SSD mobilenet v2 able to run on coreML? Will the accuracy be affected in this process?
2- Should I consider to change my neural net and retrain my data? If yes, what possibilities am I also able to run on Android? Tiny-Yolo?

**Comment by zampnrs** [comment]
After research a lot about this, I found a way to remove the cycles of **Mobilenet**, but in practice it's easier to change the neural network and retrain the data. I changed to **YOLO**, It was just needed to convert my annotation files to the YOLO format.
